src/test/resources/adfactivityrun.json

```diff
@@ -0,0 +1,35 @@
+{
+  "ActivityIterationCount": 123,
+  "ActivityName": "Activity-1",
+  "ActivityRunId": "bb41a487-309b-4d21-9ab8-2a8b948b2d18",
+  "ActivityType": "ExecutePipeline",
+  "Annotations": "[]",
+  "Category": "ActivityRuns",
+  "CorrelationId": "bb41a487-309b-4d21-9ab8-2a8b948b2d18",
+  "End": "2025-10-06T00:00:00.0000000Z",
+  "Error": "{\"errorCode\":\"InternalServerError\",\"message\":\"InternalServerError\",\"failureType\":\"SystemError\",\"target\":\"Export\",\"details\":\"\"}",
+  "ErrorCode": "InternalServerError",
+  "ErrorMessage": "InternalServerError",
+  "FailureType": "Error-1",
+  "Input": "{\"pipeline\":{\"referenceName\":\"abc_reference_one\",\"type\":\"PipelineReference\"},\"waitOnCompletion\":true,\"parameters\":{\"BillingMonth\":\"LastMonth\"}}",
+  "Level": "Error",
+  "Location": "locationcentral",
+  "OperationName": "Export - Failed",
+  "Output": "{\"pipelineName\":\"abc_reference_one\",\"pipelineRunId\":\"bb41a487-309b-4d21-9ab8-2a8b948b2d18\",\"pipelineReturnValue\":{}}",
+  "PipelineName": "Pipeline-1",
+  "PipelineRunId": "bb41a487-309b-4d21-9ab8-2a8b948b2d18",
+  "ResourceId": "/SUBSCRIPTIONS/BB41A487-309B-4D21-9AB8-2A8B948B2D18/RESOURCEGROUPS/RG-EXAMPLE-DEV-01/PROVIDERS/MICROSOFT.DATAFACTORY/FACTORIES/EXAMPLE-DEV-ABC-001",
+  "SourceSystem": "Azure",
+  "Start": "2025-10-06T00:00:00.0000000Z",
+  "Status": "Failed",
+  "Tags": "{}",
+  "Type": "ADFActivityRun",
+  "UserProperties": "{}",
+  "TimeGenerated": "2025-10-06T00:00:00.0000000Z",
+  "_ItemId": "bb41a487-309b-4d21-9ab8-2a8b948b2d18",
+  "TenantId": "bb41a487-309b-4d21-9ab8-2a8b948b2d18",
+  "_ResourceId": "/SUBSCRIPTIONS/BB41A487-309B-4D21-9AB8-2A8B948B2D18/RESOURCEGROUPS/RG-EXAMPLE-DEV-01/PROVIDERS/MICROSOFT.DATAFACTORY/FACTORIES/EXAMPLE-DEV-ABC-001",
+  "_SubscriptionId": "bb41a487-309b-4d21-9ab8-2a8b948b2d18",
+  "_TimeReceived": "2025-10-06T00:00:00.0000000Z",
+  "_Internal_WorkspaceResourceId": "/subscriptions/bb41a487-309b-4d21-9ab8-2a8b948b2d18/resourcegroups/rg-example-law-01/providers/microsoft.operationalinsights/workspaces/law-example-abc-01"
```

Comment:
this has a company name within the test data and therefore breaches Procedure: Information Security in Unit Tests, please rename such.